### PR TITLE
Improve documentation for Mautic credentials

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: docs/nodes/credentials/Mautic/README.md [hunk 1]
```diff
@@ -15,20 +15,46 @@ Create an account on a [Mautic](https://www.mautic.org/) instance.
...
+1. Open your Mautic dashboard.
+2. Click on the gear icon on the top right and select ***API Credentials***.
+3. Click on the ***+ New*** button on the top right.
+4. Select 'OAuth 2' from the ***Authorization Protocol*** dropdown list.
+5. Enter the name of the credentials in the ***Name*** field.
+6. Copy the 'OAuth Callback URL' provided in the 'Mautic OAuth2 API' credentials in n8n.
+7. Paste it in the ***Redirect URI*** field on the Mautic credentials page.
+8. Click on the ***Apply*** button.
+9. Enter the name for your credentials in the ***Credentials Name*** field in the 'Mautic OAuth2 API' credentials in n8n.
+10. Enter the authorization URL in the ***Authorization URL*** field. Refer to the [FAQs] to learn more about the authorization URL.
```
Comment (tanay1337) [inline docs/nodes/credentials/Mautic/README.md:27]: @harshil1712 There are no FAQs in this document + the link isn't there. Can you please fix?

File: docs/nodes/credentials/Mautic/README.md [hunk 2]
```diff
@@ -15,20 +15,46 @@ Create an account on a [Mautic](https://www.mautic.org/) instance.
...
+11. Enter the access toke URL in the ***Access Token URL*** field. Refer to the [FAQs] to learn more about the access token URL.
```
Comment (tanay1337) [inline docs/nodes/credentials/Mautic/README.md:28]: @harshil1712 There are no FAQs in this document + the link isn't there. Can you please fix?

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: docs/nodes/credentials/Mautic/README.md (hunk 1 above)
Comment (harshil1712) [reply docs/nodes/credentials/Mautic/README.md]: Removed the step as it is no longer needed in the newer version.

File: docs/nodes/credentials/Mautic/README.md (hunk 2 above)
Comment (harshil1712) [reply docs/nodes/credentials/Mautic/README.md]: Removed the step as it is no longer needed in the newer version.

**Pull request merged**